src/Components/FinalIdeaField/FinalIdeaField.tsx

```diff
@@ -19,35 +28,35 @@ const FinalIdeaField:React.FC = () => {
   }));
   let history = useHistory();
 
-const originalQuestion = useSelector((state:AppStore) => state.currentBrainstorm.question)
+  const originalQuestion = useSelector((state:AppStore) => state.currentBrainstorm.question);
 
   const handleChange = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
-    setBrainstormIdea(e.target.value)
-  }
+    setBrainstormIdea(e.target.value);
+  };
 
   const postBrainstormIdea = async () => {
-    const brainstorm = {
+    const brainstorm: postedBrainstorm = {
       idea: brainstormIdea,
       id,
       action: action.action,
       isGenius,
       question,
       categories
-    }
+    };
 
-    const otherBrainstorm = {
+    const otherBrainstorm: Brainstorm = {
       response: brainstormIdea,
       id: Date.now(),
       action: action,
       isGenius,
```

Comment:
Thank you for refactoring the Brainstorm interface. I am wondering if we may need to revisit this when we go to make an edit to an existing brainstorm, that was recently created because the id (Date.now) will conflict with what id gets assigned by the database. We will need to work with the backend and may need them to refactor how the response is structured after a new Brainstorm is posted.

Reply:
This implementation was my doing originally, just a quickly thrown together fix to workaround. We could alternatively make another apiCall and dispatch to the store, but it feels easier to just get the response from the posting apiCall and then dispatch that to the store.